src/routes/boot.ts

```diff
@@ -384,6 +384,8 @@ const getUser = (con: DataSource, userId: string): Promise<User | null> =>
       'cover',
       'experienceLevel',
       'language',
+      'followingEmail',
+      'followNotifications',
```

Comment:
I was thinking we could maybe start moving some of these out of boot and into a separate query. We don't really need all of these at boot

Reply:
Yeah these for sure we only need only on the specific settings screen right?
Don't we do a separate query there already maybe?

Reply:
I think settings still look at boot, but @omBratteng can check that, I agree with moving them out but we are waiting for product to define new notif settings feature.

Reply:
It still uses boot